gazebo_ros/src/node.cpp

```diff
@@ -97,6 +102,17 @@ Node::SharedPtr Node::Get(sdf::ElementPtr sdf)
       parameter_sdf = parameter_sdf->GetNextElement("parameter");
     }
   }
+  // set full node name
+  full_name = ns + "/" + name;
+
+  // check if node with the same name exists already
+  if (static_node_lookup_.is_node_name_in_set(full_name)) {
+    RCLCPP_ERROR(
+      internal_logger(),
+      "Found multiple nodes with same name: %s. This is due to different plugins with same name, "
+      "either change the plugin name or use a unique namespace", full_name.c_str());
```

Comment:
This *might* be due to nodes with same name, as cases like inheritance can also cause this error, we might want to include that in this message